Tasks-Maps/src/Task_3/AnagramOfName.java

```diff
@@ -0,0 +1,43 @@
+package Task_3;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class AnagramOfName {
+    public static void main(String[] args) {
+
+    }
+
+    /*  3. Анаграмма слова x - слово, по длине равное x и состоящее из таких же букв, что и x.
+         Например, слово “vani” является анаграммой слова “ivan”, а слова “naan” и “anan” являются анаграммами слова “anna”.
+         Дан словарь, содержащий анаграммы, например {“anna”, “ivan”, “naan”, “vani”, “piotr”, “nana”, “navi”}.
+         Написать функцию, которая по слову из словаря вернет все анаграммы этого слова, которые есть в словаре.
+         Например, по слову “ivan” функция вернет {“navi”, “vani”}
+*/
+
+    public static char[] getSortedCharArrayFromString(String string) {
+        char[] charArr = string.toCharArray();
+        Arrays.sort(charArr);
+        return charArr;
+    }
+
+
+    public static List<String> getListOfAnagramsByName(List<String> anagrams, String name) {
```

Comment:
I would expect you to use either `createAnagramMap()` or its result (map) in this method.